### PR TITLE
Offer NGSI-LD in the issue page's export list

### PR DESCRIPTION
**Pull request by dkastl**
The issue page had two export lists: Redmine's own `p.other-formats` at the
bottom ("Also available in: PDF, GeoJSON, Atom") plus a second line of the same
class below the description, added by this plugin, reading "Also available as:
NGSI-LD". NGSI-LD is just another export format, so it now sits with the others.

Before: two lists, the plugin's one below the description.
After: `Also available in: PDF  GeoJSON  Atom  NGSI-LD`

## How

The core list has no view hook, so this is a Deface override anchored on the
Atom link, the same mechanism `redmine_gtt` uses to add GeoJSON (which anchors
on PDF, hence the ordering). Overrides are `require`d from `init.rb` and hidden
from Zeitwerk, again following `redmine_gtt`. The `view_issues_show_description_bottom`
hook and the `_entity_link` partial are gone, as is the now-unused
`label_fiware_entity_link` string in all three locales.

## Verification

`test/functional/issue_other_formats_test.rb` asserts the link inside
`p.other-formats`, and its absence when no instance identifier is set (there is
no representation to serve then, and `FiwareEntitiesController` 404s). It is an
integration test on purpose: a stale CSS selector would silently insert nothing
and a unit test would not notice. Confirmed it does catch that, by pointing the
selector at a non-existent element and watching it fail.

Full suite: 301 runs, 1092 assertions, 0 failures. On the dev instance the
rendered list is `PDF GeoJSON Atom NGSI-LD` with exactly one `p.other-formats`
on the page, and the link returns the entity as `application/ld+json`.

### CODE REVIEW

File: init.rb [hunk 1]
```diff
@@ -53,6 +53,13 @@
   end
 end
 
+# Deface overrides are not autoloaded (they define no constants), so they are
+# required here and hidden from Zeitwerk, the same way redmine_gtt does it.
+Dir.glob("#{Rails.root}/plugins/redmine_gtt_fiware/app/overrides/**/*.rb").each do |path|
+  Rails.autoloaders.main.ignore(path)
+  require path
+end
```
[the remark this answers is not in the document]
Comment (dkastl) [reply init.rb]: Fixed in 96eec0b: the glob now resolves from `__dir__`, so neither `Rails.root` nor the folder name matters.

Worth noting the practical risk was smaller than it looks, because Redmine itself already assumes the id-named folder: `Redmine::Plugin#directory` is derived from the registered plugin id, not from where the code actually sits, so a renamed folder breaks the plugin view path and locale loading before it gets anywhere near the overrides. The fix is free either way and one less thing to keep in sync.

File: test/functional/issue_other_formats_test.rb [hunk 2]
```diff
@@ -0,0 +1,35 @@
...
+  end
+
+  def test_lists_ngsi_ld_among_the_other_formats
+    with_settings plugin_redmine_gtt_fiware: INSTANCE_SETTINGS do
+      get "/issues/#{@issue.id}"
+    end
+    assert_response :success
+    assert_select 'p.other-formats' do
+      assert_select "a[href=?]", "/fiware/issues/#{@issue.id}/entity", text: 'NGSI-LD'
+    end
```
[the remark this answers is not in the document]
Comment (dkastl) [reply test/functional/issue_other_formats_test.rb]: Good catch, fixed in 96eec0b. This was the one assertion actually worth having and it was missing: the test as written passed with the old duplicate line still present, so it did not cover the regression the change is about. Both tests now assert `count: 1`.

Verified it has teeth by putting the removed `view_issues_show_description_bottom` hook back and re-running: `Expected exactly 1 element matching "p.other-formats", found 2`.
